Enable Python 3.12

duncanmmacleod opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

duncanmmacleod committed:
enable Python 3.12

duncanmmacleod commented:
This is blocked on getting a new release of pycbc for Python 3.12, which is imminent.

duncanmmacleod committed:
Minor recipe housekeeping

duncanmmacleod committed:
Depend on astropy-base instead of astropy

duncanmmacleod committed:
Add @duncanmmacleod as co-maintainer

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I was trying to look for recipes to lint for you, but it appears we have a merge conflict. Please try to merge or rebase with the base branch to resolve this conflict.

Please ping the 'conda-forge/core' team (using the `@` notation in a comment) if you believe this is a bug.

duncanmmacleod committed:
MNT: Re-rendered with conda-smithy 3.53.3 and conda-forge-pinning 2025.12.01.21.34.22

Other tools:
- conda-build 25.9.0
- rattler-build 0.48.1
- rattler-build-conda-compat 1.4.9

duncanmmacleod committed:
MNT: Re-rendered with conda-smithy 3.53.3 and conda-forge-pinning 2025.12.01.21.34.22

Other tools:
- conda-build 25.9.0
- rattler-build 0.48.1
- rattler-build-conda-compat 1.4.9

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

duncanmmacleod commented:
@a-renzini, please consider this change. Note that this adds me as a maintainer of the recipe, this is mainly to help with housekeeping/migrations like this.
